tests/data/carbon_setup.json

```diff
@@ -1,5 +1,6 @@
 {
-  "sessionID": "carbon_setup",
+  "setupHash": "abfd5c3da293693a546c",
```

Comment:
I might have missed it, but what's this `setupHash`?

Reply:
Oops, this was an early version. I changed to `setupKey` in the end. I'll update this test data, or maybe see about generating it within the test itself.